Lib/http/server.py

```diff
@@ -909,6 +956,32 @@ def guess_type(self, path):
             return guess
         return 'application/octet-stream'
 
+    def parse_range(self):
+        """Return a tuple of (start, end) representing the range header in
+        the HTTP request. If the range header is missing, not resolvable,
+        or trivial (namely "byte=-"), this returns None.
```

Comment:
```suggestion
        or trivial (namely "bytes=-"), this returns None.
```

Sorry, there was a typo in my suggestion.

Reply:
I wouldn’t say “Range: bytes=-” is trivial; it is not a valid byte range according to the spec.

Reply:
> it is not a valid byte range according to the spec

I agree with this, I'm going to categorize “bytes=-” as not resolvable.

Reply:
Oh right. Sorry for this (I assumed that it would be trivial but it's not valid indeed). You can revert to the previous comment!